src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/OnepageSuccessAction.php

```diff
@@ -49,6 +49,14 @@ public function execute(Varien_Event_Observer $observer)
                 $pipelineRequestBlock->setTemplate('shopgate/cloudapi/pipelineRequest.phtml');
                 $layout->getBlock('head')->addJs('shopgate/pipelineRequest.js');
                 $layout->getBlock('head')->append($pipelineRequestBlock);
+
+                /** @var Shopgate_Cloudapi_Block_AnalyticsLogPurchase  $analyticsLogPurchaseBlock */
```

Comment:
this pertains to the PR I did not review, adjust ```->getOrderIds()``` to use ```getData('order_ids')``` instead to avoid the warning.
Secondly, use this to avoid warnings of header setting:
```
/** @var Mage_Page_Block_Html_Head $head */
$head = $layout->getBlock('head');
$head->addJs('shopgate/pipelineRequest.js');
...
```